tdrs-frontend/nginx/cloud.gov/buildpack.nginx.conf

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
Before merging, I would want to deploy-to/test-in prod and would want to coordinate this with @ADPennington and setup maintenance window w/ UX